packages/manager/apps/hycu/src/pages/dashboard/general-information/BillingInformations/BillingInformationsTile.tsx

```diff
@@ -100,17 +107,31 @@ const BillingInformationsTile = ({ serviceName }: { serviceName: string }) => {
           value: (
             <div className="flex flex-col gap-4">
               <div>
-                {isLoading
-                  ? Array.from({ length: 3 }).map((_, index) => (
-                      <OsdsSkeleton key={index} />
-                    ))
-                  : serviceDetails?.data.customer.contacts.map((contact) => (
+                {isLoading ? (
+                  Array.from({ length: 3 }).map((_, index) => (
+                    <OsdsSkeleton key={index} />
+                  ))
```

Comment:
```suggestion
                    <>
                      <OsdsSkeleton />
                      <OsdsSkeleton />
                      <OsdsSkeleton />
                    </>
                  )
```

No need to re-calculate a list of static components on each render I think

Reply:
I think the loop is better to maintain the code and avoid repetition. Maybe another opinion to known what method to choose ?